packages/design-system/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { forwardRef, HTMLAttributes, ReactElement, Ref } from 'react';
+import { useTranslation } from 'react-i18next';
+import classnames from 'classnames';
+
+import styles from './Breadcrumbs.module.scss';
+import Link from '../Link';
+import Dropdown from '../Dropdown/Dropdown';
+import { ButtonTertiary } from '../Button';
+import { StackHorizontal } from '../Stack';
+import Divider from '../Divider';
+import VisuallyHidden from '../VisuallyHidden';
+
+type BreadcrumbsLink = {
+	label: string;
+	href: string;
+	target?: string;
+};
+
+type BreadcrumbsRouterLink = {
+	label: string;
+	target?: string;
+	as: ReactElement;
+};
+
+type BreadcrumbsItem = (BreadcrumbsRouterLink | BreadcrumbsLink)[];
+
+type BreadCrumbsProps = Omit<HTMLAttributes<HTMLElement>, 'className' | 'style'> & {
+	items: BreadcrumbsItem;
+};
+
+const maxBreadcrumbsItemLength = 4;
+
+const Breadcrumbs = forwardRef(({ items, ...rest }: BreadCrumbsProps, ref: Ref<HTMLElement>) => {
+	const { t } = useTranslation();
+	const buildEntries = () => {
+		if (items.length > maxBreadcrumbsItemLength) {
+			const origin = items[0];
+			const suffix = items.slice(-2);
+			const collapsed = items.slice(1, items.length - 2);
+
+			return (
+				<>
+					<li className={styles.entry}>
+						<StackHorizontal gap="S" align="center" wrap="nowrap">
+							{'href' in origin ? (
+								<Link href={origin.href} target={origin.target} withEllipsis>
+									{origin.label}
+								</Link>
+							) : (
+								<Link as={origin.as} target={origin.target} withEllipsis>
+									{origin.label}
+								</Link>
+							)}
```

Comment:
```suggestion
						<Link as={origin.as} href={origin.href} target={origin.target} withEllipsis>
							{origin.label}
						</Link>
```

Reply:
No. You cannot have both `as` and `href`, they are exclusive typings.

Reply:
Well it's precisely because `Link` is typed to let you know that you must either pick `href` or `as` that you need to refine your type here. 
I don't see it as duplication: today they are the same component, but they are two different branches of the API. We could easily split it.